eradiate/radprops/rad_profile.py

```diff
@@ -37,6 +37,7 @@
 import xarray as xr
 
 import eradiate
+import eradiate.data as data
```

Comment:
Please favour relative imports in lib code (as opposed to test code):

```suggestion
from .. import data
```